src/services/analytics.ts

```diff
@@ -50,11 +52,42 @@ export function onAnalyticsUpdate() {
 }
 
 
-
 async function __getData() {
+
   try {
-    const data = await API.graphql(graphqlOperation(listRaamalytics)) as any;
+    if(token == null){
+      const data = await API.graphql(graphqlOperation(listRaamalytics)) as any;
+     // console.log(data);
+      token  = data.data.listRaamalytics.nextToken;
+
+      data.data.listRaamalytics.items.forEach((element:any) => {
+        element["predicted_arrival_time"] = moment(element["predicted_arrival_time"]).unix();
+        element["wind_speed_plus_2hr"] = parseFloat(element["wind_speed_plus_2hr"]);
+      });
+
+      console.log(data.data.listRaamalytics.items);
```

Comment:
should remove all console.log from this code so it doesn't pollute the console when you're not actively developing it